python/across_api/across/hello.py

```diff
@@ -33,10 +33,8 @@ def __init__(self, name: Optional[str] = None):
             Your name
         """
         self.name = name
-        if self.validate_get():
-            self.get()
 
-    def get(self) -> bool:
+    async def get(self) -> bool:
```

Comment:
Functions that don't contain any `await` statements need not be `async`.

Reply:
FastAPI documentation says it's better to use async on simple compute functions.

https://fastapi.tiangolo.com/async/#path-operation-functions

Reply:
How very curious. Thanks for explaining!